### DISCUSSION TITLE
Modify cluster options for PBS via dsub

### DISCUSSION THREAD

**Discussion by atongsa** [nextflow-io/nextflow#1921] [Q&A]
## question_description
- my platform is torque;
- but the HPC use dsub to submite jobs, so i cant submite jobs use nextflow default qsub;
- and also, there is a line in submitting jobs, like below, which is not print by nextflow torque api;
```
#PBS -j oe
#HSCHED -s hschedd
```
## my solution
- write a submite script and apply resources, and put nextflow run in the script, config nextflow to processor.local
- will this method use HPC resources corretly ?

**Comment by abhi18av** [comment]
Hi @atongsa ,

Perhaps the `clusterOptions` process directive described [here](https://www.nextflow.io/docs/latest/process.html#clusteroptions) might be of use.

It could be used as shown below


```
clusterOptions  '#HSCHED -s hschedd'


```

**Comment by atongsa** [comment]
> Hi @atongsa ,
> 
> Perhaps the `clusterOptions` process directive described [here](https://www.nextflow.io/docs/latest/process.html#clusteroptions) might be of use.
> 
> It could be used as shown below
> 
> ```
> clusterOptions  '#HSCHED -s hschedd'
> ```

thanks @abhi18av  , what anout `dsub`, my cluster use `dsub not qsub with torque`, so i can't submite jobs with nextflow

**Comment by abhi18av** [comment]
@atongsa , unfortunately I can't comment that one. 

Though the docs mention `qsub` in the [executors](https://www.nextflow.io/docs/latest/executor.html?highlight=qsub#pbs-torque) docs.

**Comment by atongsa** [comment]
> @atongsa , unfortunately I can't comment that one.
> 
> Though the docs mention `qsub` in the [executors](https://www.nextflow.io/docs/latest/executor.html?highlight=qsub#pbs-torque) docs.

thank you @abhi18av
